Detectors/MUON/MCH/Workflow/src/cru-page-to-digits-workflow.cxx

```diff
@@ -0,0 +1,39 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file    cru-page-to-digits-workflow.cxx
+/// \author  Andrea Ferrero
+///
+/// \brief This is an executable that runs the decoding via DPL.
+///
+/// This is an executable that takes a raw buffer from the Data Processing Layer, runs the decoding and sends the digits via the Data Processing Layer.
```

Comment:
I guess this is a cut and paste from the raw-to-digits workflow.
BTW, please add in the README some explanation about the difference about the two

Reply:
See above.